src/scss/03_molecules/form/_form-input-group.scss

```diff
@@ -0,0 +1,65 @@
+.form-input-group {
+  flex: 1;
+  position: relative;
+
+  @include mq(small) {
+    flex: 0 0 auto;
+  }
+}
+
+.form-input-group .form-field__error {
+  flex: 1;
+  padding-left: 0;
+}
+
+.form-input-group__append,
+.form-input-group__prepend {
+  align-items: center;
+  display: flex;
+  height: get($formFieldStyleConfig, height);
+  justify-content: center;
+  mix-blend-mode: darken;
+  position: absolute;
+  width: get($formFieldStyleConfig, height);
+  z-index: 2;
+
+  &.is-large {
+    width: #{uGet($formFieldStyleConfig, height) * 1.5}rem;
+  }
+}
+
+.form-input-group__append {
+  border-left: get($formFieldStyleConfig, borderWidth) solid get($formFieldStyleConfig, borderColor);
+  cursor: pointer;
+  right: 0;
+
+  & ~ .form-input {
+    padding-right: #{uGet($formFieldStyleConfig, height) + uGet($formFieldStyleConfig, paddingLeft)}rem;
+  }
+
+  &.is-large ~ .form-input {
+    padding-right: #{uGet($formFieldStyleConfig, height) * 1.5 + uGet($formFieldStyleConfig, paddingLeft)}rem;
+  }
+
+  & ~ .form-input:disabled {
```

Comment:
` & ~ .form-input:disabled` -->  `& ~ .form-input[disabled],  & ~ .form-input.is-disabled`